.github/workflows/graphviz.yml

```diff
@@ -0,0 +1,18 @@
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo content
+        uses: actions/checkout@v2 
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
```

Comment:
Thank you for the graphviz support!
Ideally if python3.8 can be supported, that would be great; as python3.6 is not updated by Python Foundation since Dec 2021.
Currently, the following error will occur when using python3.8:
src/checkdep_freetype2.c(1): fatal error C1083: Cannot open include file: 'ft2build.h': No such file or directory
When running
```
python3 setup.py install
```

Reply:
Done